services/efs-guardian/src/simcore_service_efs_guardian/services/efs_manager.py

```diff
@@ -1,10 +1,14 @@
+import os
 from dataclasses import dataclass
 from pathlib import Path
 
 from fastapi import FastAPI
 from models_library.projects import ProjectID
 from models_library.projects_nodes_io import NodeID
 
+_EFS_LINUX_USER = 8006
+_EFS_GROUP_LINUX_GROUP = 8106
+
```

Comment:
Suggestion: the SC_USER_ID is an ENV variable. I think you might even have it already in the settings. you should use it instead of hard-coding here.
And do the same with the group if you need it.

Reply:
I also agree with using env vars/args for these values.

Reply:
agree as well

Reply:
done